tensorboard/BUILD

```diff
@@ -62,6 +63,14 @@ ts_web_library(
 )
 
 py_library(
+    name = "expect_mock_installed",
```

Comment:
It just occurred to me that mock is available as tf.test.mock. Replaced this with that. Done.